src/main/resources/application-dev.yml

```
@@ -3,4 +3,13 @@ cors:
-    access-token: ${LINE_ACCESS_TOKEN_DEV}
\ No newline at end of file
+    access-token: ${LINE_ACCESS_TOKEN_DEV}
+
+  aws:
+    s3:
+      image-bucket:
+        name: dev-bucket
+      region: ap-northeast-1
+      credentials:
+        access-key: dev-access-key
+        secret-key: dev-secret-key
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)